File: portal_2.API/Models/Broker.cs
using System;
using System.Collections.Generic;

namespace portal_2.API.Models
{
    public class Broker
    {
        public int BrokerId { get; set; }
        public int EntityId { get; set; }
        public string Username { get; set; }
        public byte[] PasswordHash { get; set; }
        public byte[] PasswordSalt { get; set; }
        public string Fstname { get; set; }
        public string Lstname { get; set; }
        public int BranchId { get; set; }
        public int RbmId { get; set; }
        public int AffiliateId { get; set; }
        public string Address { get; set; }
        public string City { get; set; }   
        public int StateId { get; set; }
        public int ZipCode { get; set; }
        public string Email { get; set; }  
        public string AltEmail { get; set; }
        public int Phone { get; set; }
        public int PhoneExt { get; set; }
        public string AltPhone { get; set; }
        public bool IsContracted { get; set; }
        public int NpnId { get; set; }
        public DateTime ModifyWhen { get; set; }
        public DateTime LastActive { get; set; }
        public ICollection<App> Apps { get; set; }
    }
}

File: portal2-SPA/src/app/_models/broker.ts
import { App } from './App';

export interface Broker {
    id: number;
    brokerTempId: number;
    username: string;
    fstname: string;
    lstname: string;
    branch: string;
    address: string;
    city: string;
    state: string;
    ZipCode: number;
    email: string;
    phone: number;
    isContracted: boolean;
    modifyWhen: Date;
    lastActive: Date;
    phoneExt?: number;
    AltPhone?: number;
    altEmail?: string;
    rbm?: string;
    affiliate?: string;
    npnId?: number;
    entityId: number;
    apps?: App[];

}
